go/test/endtoend/vreplication/fk_ext_test.go

```diff
@@ -350,13 +339,9 @@ func moveKeyspace(t *testing.T) {
 
 func newKeyspace(t *testing.T, keyspaceName, shards, vschema, schema string, tabletId, numReplicas int) map[string]*cluster.VttabletProcess {
 	tablets := make(map[string]*cluster.VttabletProcess)
-	cellName := fkextConfig.cell
 	cell := vc.Cells[fkextConfig.cell]
+	vtgate := cell.Vtgates[0]
```

Comment:
Another place that IMO is worth a nil check. I'm thinking that tracking down issues when there's a panic will be much harder than a NotNil check with a helpful error message (with file and line number).

Reply:
We error out right at cluster setup if we are unable to create a vtgate for a cell.